dace/dtypes.py

```diff
@@ -599,6 +599,7 @@ def from_json(json_obj, context=None):
 
         try:
             typeclass = json_to_typeclass(json_obj['ctype'], context)
+            return typeclass()
```

Comment:
This is part of a greater fix. I identified the issue by having a callback to the `print` method, which is registered on the SDFG as a `dtypes.pyobject` Scalar. When reading the SDFG from file, it becomes `dtypes.opaque`. Calling the program with loaded SDFG subsequently fails, because the opaque type is missing methods needed by ctypes. The return statement is needed because `opaque(pyobject)` is opaque, not pyobject. You can comment out the line and run the new test to see the exact ctypes error.